.github/workflows/auto-labeling.yml

```diff
@@ -0,0 +1,14 @@
+on:
+  schedule:
+    # Run this action every 10 minutes
+    - cron: "*/10 * * * *"
```

Comment:
To apply labels on an issue or a pull request, a write permmision is required, but the GitHub token created for a pull request coming from a forked repo doesn't have a write permission. Instead, we can use the `schdule` event and trigger this action on a certain interval.